### PR TITLE
#213 TMP dir is also created and logged at start.

### PR DESCRIPTION
**Pull request by kozaka-tv**

### CODE REVIEW

**Comment by skully** [comment]
200w vs. asterisk :)


Made a small test script for paths:
```
(venv) PS C:\Users\dphdp> cat .\tmp2\path.py
import sys
import os

def print_call_info():
        # Location of the program being executed
    program_location = os.path.abspath(sys.argv[0])

    # Location from which the program was called
    call_location = os.getcwd()

    print(f"Call location: {call_location}")
    print(f"Program location: {program_location}")

print_call_info()

(venv) PS C:\Users\dphdp> .\tmp2\path.exe
Call location: C:\Users\dphdp
Program location: C:\Users\dphdp\tmp2\path.exe
(venv) PS C:\Users\dphdp>
```

File: definitions.py [hunk 1]
```diff
@@ -1,13 +1,10 @@
 import os
 
-TMP_DIR_NAME = "tmp"
-CACHE_DIR_NAME = "cache"
-PSARC_INFO_FILES_DIR_NAME = "psarc-info-files"
+ROOT_DIR = os.getcwd()
```
Comment (skully) [inline definitions.py:3]: This is the referred line in my comment

**Comment by kozaka-tv** [comment]
Changed to:

`ROOT_DIR = os.path.dirname(os.path.abspath(sys.argv[0]))`

image

**Review by skully — APPROVED** [review]
Works for me

**Pull request merged**